app/src/main/res/layout-land/fragment_settings.xml

```diff
@@ -0,0 +1,123 @@
+                android:layout_below="@+id/profile_image_settings"
+                android:layout_marginStart="@dimen/space_130"
+                android:layout_marginTop="-23dp"
+                android:src="@drawable/edit" />
+        </RelativeLayout>
+
+        <androidx.cardview.widget.CardView
+            android:id="@+id/settings_info_card"
+            android:layout_width="@dimen/space_350"
+            android:layout_height="wrap_content"
+            app:cardCornerRadius="@dimen/space_20"
+            android:layout_gravity="bottom|end"
+            android:layout_marginBottom="@dimen/space_10"
+            android:layout_marginEnd="@dimen/space_20"
+            android:layout_marginTop="@dimen/space_10"
+            android:background="@color/light_grey">
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical"
+                android:padding="@dimen/space_10">
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="horizontal">
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:text="@string/title_about"
+                        android:textSize="@dimen/text_18"
+                        android:textColor="@color/black"/>
+                    <ImageView
+                        android:id="@+id/edit_about_info"
+                        android:layout_width="@dimen/space_18"
+                        android:layout_height="@dimen/space_18"
+                        android:src="@drawable/edit"
+                        android:layout_marginStart="@dimen/space_20" />
+                </LinearLayout>
+
+                <TextView
+                    android:id="@+id/about_settings"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="@string/about_text_settings"
+                    android:textSize="@dimen/text_18"
+                    android:layout_marginTop="@dimen/space_10"/>
+                <Button
+                    android:id="@+id/facebook_settings"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginTop="@dimen/space_20"
+                    android:background="@color/purple_700"
+                    android:text="My Facebook link"
+                    android:textColor="@color/white"
+                    android:textSize="@dimen/space_16" />
+            </LinearLayout>
+        </androidx.cardview.widget.CardView>
+    </androidx.cardview.widget.CardView>
+
+
+</androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
Side note: 
If you look into PR you can find the "red minus" sign at the end of files like in this one.
A common practice is that the file should end with one empty line. That's why GitHub also remind these with these warning sign.